tilelang/language/v2/builder.py

```
@@ -911,10 +958,38 @@ def _parse_phase1_key(self, *args, **kwargs):
+
+        Eager style functions use the builder pattern with type annotations:
+            @jit
+            def foo(A, B):
+                A: T.Tensor[...]
+                with T.Kernel(...): ...
+                # no return
+        """
+        try:
+            result = self.orig_func(*args, **kwargs)
+            return isinstance(result, PrimFunc)
+        except Exception:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)